unit-tests/live/frames/test-fps.py

```diff
@@ -62,8 +62,8 @@ def frame_cb(frame):
 
 
 delta_Hz = 1
-tested_fps = [6, 15, 30, 60, 90]
```

Comment:
Did you try running the nightly and verify there are no timeouts?

I believe we don't have models that support both 5 and 6 FPS but this test is long and we should verify no extra time is needed

Reply:
WIP will update

Reply:
Passed - LRS_libci_pipeline-9992